src/topology/omega_complete_partial_order.lean

```diff
@@ -0,0 +1,155 @@
+/-
+Copyright (c) 2020 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Simon Hudon
+-/
+import topology.basic
+import order.omega_complete_partial_order
+
+/-!
+# Scott Topological Spaces
+
+I don't know what they're good for but their notion
+of continuity is equivalent to continuity in ωCPOs.
```

Comment:
```suggestion
A type of topological spaces whose notion
of continuity is equivalent to continuity in ωCPOs.
```

Reply:
I forgot that was the module comment I wrote :P I was assuming that would be the first thing reviewers would harp on.